_includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md

```diff
@@ -656,6 +674,53 @@ class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
 }
 ```
 
+#### Register sales plugins
```

Comment:
Please move adjustments below to separate IG - `Service Points + Order Management feature`

Reply:
And transfers, modules above not? Just wondering while plugins depend on these. 🤔

Reply:
no, modules and transfers should be located in the main feature.